delivery_package_number/wizard/stock_inmediate_transfer.py

```diff
@@ -9,6 +9,13 @@ class StockImmediateTransfer(models.TransientModel):
     number_of_packages = fields.Integer(
         help="Set the number of packages for this picking(s)",
     )
+    ask_number_of_packages = fields.Boolean(compute="_compute_ask_number_of_packages")
+
+    def _compute_ask_number_of_packages(self):
+        for item in self:
+            if len(item.pick_ids) != 1:
+                item.ask_number_of_packages = False
+            item.ask_number_of_packages = bool(item.pick_ids.carrier_id)
```

Comment:
If there's more than one picking, no error will happen as they are joined together in a recordset

```suggestion
            item.ask_number_of_packages = bool(item.pick_ids.carrier_id)
```

Reply:
The thing is to avoid showing the package number in those cases as well.

Reply:
Yes, and that's the case for my suggestion.